src/app/node-data/basic/provider/kubevirt/template.html

```diff
@@ -172,64 +172,80 @@
 
   <div *ngIf="secondaryDisksFormArray.length === 0"
        class="no-secondary-disks-msg km-text-muted">
-    No secondary disks configured. Use the add button above to configure up to three disks.
+    Optional disks that will be mounted to the nodes. Configure up to three disks.
   </div>
 
-  <mat-card-header class="km-no-padding">
-    <mat-card-title>Affinity Settings</mat-card-title>
-  </mat-card-header>
-
-  <ng-container *ngTemplateOutlet="selectAffinityPreset; context: {label: 'Pod Affinity Preset', controlName: Controls.PodAffinityPreset, resetCallback: resetPodAffinityPresetControl.bind(this)}"></ng-container>
-
-  <ng-container *ngTemplateOutlet="selectAffinityPreset; context: {label: 'Pod Anti-Affinity Preset', controlName: Controls.PodAntiAffinityPreset, resetCallback: resetPodAntiAffinityPresetControl.bind(this)}"></ng-container>
-
-  <ng-container *ngTemplateOutlet="selectAffinityPreset; context: {label: 'Node Affinity Preset Type', controlName: Controls.NodeAffinityPreset, resetCallback: resetNodeAffinityPresetControl.bind(this)}"></ng-container>
+  <km-expansion-panel expandLabel="ADVANCED SCHEDULING SETTINGS"
```

Comment:
IMO its not necessary because we already have this up/down icons which is self explanatory.

![screenshot-localhost_8000-2022 08 12-16_16_12](https://user-images.githubusercontent.com/13975988/184343426-25942059-d5db-4499-aa93-b53b3cf57734.png)